scripts/compose.py

```diff
@@ -1758,6 +1758,17 @@ def start_handler(self):
         services = {}
         for service in selections:
             services.update(service.render())
+
+        # expose a list of enabled opbeans services to all opbeans services. This allows them to talk amongst each other
+        # and have a jolly good distributed time
+        enabled_opbeans_services = [k for k in services.keys() if k.startswith("opbeans-") and not k in ("opbeans-rum", "opbeans-load-generator")]
```

Comment:
maybe we want only to enable DT on some test, we could add a parameter for that something like this

if options.get('opbeans_dt_enable'):

Reply:
how about we kill two birds with one stone* and use a `--opbeans-dt-probability` float flag? Setting it to 0 disables dt, default will be 0.5

* no birds have been killed during the production of this comment

Reply:
@kuisathaverat implemented in 4d1df35